input/data/2014/2014A%26A...562A..40H/tev-000117.yaml

```diff
@@ -1,4 +1,23 @@
 source_id: 117
 reference_id: 2014A&A...562A..40H
 
-# TODO: add data
+data:
+  livetime: 120
+  significance: 8.2
+  excess: 117
+
+pos:
+  ra: {val: 18h18m4.8s, err: 3.1s, err_sys: }
+  dec: {val: -15d28m1s, err: 43s, err_sys: }
+
+morph:
+  type: gauss
```

Comment:
> The 99% confidence upper limit on the Gaussian width of the source is 0.072◦. No significant ellipticity of the source was found.

Please put this upper limit for `sigma` like in the other case where we had an upper limit on extension earlier today.